HISTORY.md

```diff
@@ -1,12 +1,4 @@
-## Version 5.6.0
```

Comment:
5.6.0 is removed and is generated by github-changelog-generator to workaround https://github.com/skywinder/github-changelog-generator/issues/555